README.md

```diff
@@ -0,0 +1,22 @@
+# Arizona Bootstrap
+
+Build responsive, mobile-first projects on the web with the University of Arizona's theme for Bootstrap 4.
+
+Bootstrap is an open source toolkit for developing with HTML, CSS, and JS. Quickly prototype your ideas or build your entire app with our Sass variables and mixins, responsive grid system, extensive prebuilt components, and powerful plugins built on jQuery.
+
+A team of web-focused volunteers known as Arizona Digital meets weekly to build and test products like Arizona Bootstrap and Arizona Quickstart. Contributions are what keep Arizona Bootstrap moving and improving, and are in everyone's best interest.
+
+<!-- URL doesn't exists yet -->
+[Explore Arizona Bootstrap Docs](digital.arizona.edu/arizona-bootstrap)
+
+## Contributing
+
+Please refer to the [Contributing doc](./contributing.md).
+
+## Arizona Bootstrap vs. UA Bootstrap
+
+Arizona Bootstrap is built off of Bootstrap 4, and UA Bootstrap is built off of Bootstrap 3.
+
+<!-- URLs doesn't exist yet -->
+- [Learn about migration from Bootstrap 3 to Bootstrap 4](digital.arizona.edu/arizona-bootstrap/master/docs/0.0/migration)
```

Comment:
The wording on these links might be confusing.

- [Learn about migration from Bootstrap 3 to Bootstrap 4](digital.arizona.edu/arizona-bootstrap/master/docs/0.0/migration)
Might be,  - [Learn about migration from UA Bootstrap to Arizona Bootstrap](digital.arizona.edu/arizona-bootstrap/master/docs/0.0/migration)

Reply:
So the Bootstrap 3 vs. 4 is the migration page, and the UA Bootstrap vs. Arizona Bootstrap is more specific to our overrides to vanilla Bootstrap, and how that translates into migration. Is that what you mean?